### ISSUE TITLE
looking for new owner of repo

### ISSUE THREAD

**Issue by jeroenterheerdt** [jeroenterheerdt/ring-hassio#72]
Hi all, first of all, sorry for the silence from my side. I moved and have since left behind the Ring eco system, so I don't own a Ring any more. I can archive this repo, which I will, if I don't find any new owner to transfer this to in the next couple weeks.
Please let me know if you're interested in becoming the new owner.

**Comment by minasolution** [comment]
@jeroenterheerdt out of curiosity, what was the new eco that you moved to? 
I'm looking for alternates to my Ring too and want it to work nicely with HA. Many thanks !

**Comment by jeroenterheerdt** [comment]
haven't found one yet, but I think I'll move to a tuya based one, since I am pretty happy with localtuya.

**Comment by minasolution** [comment]
Ok thank you. Tuya have a nice history of countering the idea of using their devices locally so avoid them as plague!

Local Tuya is nice at this time of writing... until Tuya decides to block it in the future. Then all of our money will be wasted. 

This is because we cannot block them forever. (e.g. if the device drops/resets, then we have to grant it internet access to re-add. Tuya will take this chance to update it to totally prevent it from working if no internet access).

**Comment by jeroenterheerdt** [comment]
yeah, forget about tuya for doorbells, localtuya doesn't even support doorbells. Anyway, I think I might go with Eufy: https://www.amazon.com/dp/B0868LH58W/?coliid=IKK2CBE3K9PH6&colid=2NWX8UG1FRX0J&psc=1&ref_=lv_ov_lig_dp_it. The custom component looks really good and stable: https://github.com/fuatakgun/eufy_security
